config/config.yml

```diff
@@ -115,28 +115,28 @@ projects:
     additional_parameters:
       TASKCLUSTER_CLIENT_ID: project/autophone/gecko-t-bitbar-unit-g5
       TC_WORKER_TYPE: gecko-t-bitbar-gw-unit-g5
-      DOCKER_IMAGE_VERSION: 20190408T165211
+      DOCKER_IMAGE_VERSION: 20190423T121205
   # used for building new docker images
   mozilla-docker-build:
     device_group_name: motog4-docker-builder
     device_model: motog4
     framework_name: Docker image build
     description: Mozilla Docker image build
-    test_file: mozilla-docker-20190208T112604.zip
+    test_file: mozilla-docker-20190423T121205.zip
     application_file: aerickson-Testdroid.apk
     additional_parameters:
-      DOCKER_IMAGE_VERSION: 20190208T112604
+      DOCKER_IMAGE_VERSION: 20190423T121205
   # used for testing new docker images
-  mozilla-docker-image-test:
-    device_group_name: motog5-test
-    device_model: motog5
```

Comment:
Want to put this back in the pool now that you are disabling the mozilla-docker-image-test project?